test/com/google/enterprise/adaptor/prebuilt/PrebuiltTransformsTest.java

```diff
@@ -706,20 +708,23 @@ public void testDeleteToString() {
 
   @Test
   public void testDeleteToStringWithKeys() {
-    Map<String, String> config = new HashMap<String, String>();
+    Map<String, String> config = new LinkedHashMap<String, String>();
     config.put("key2", "colour");
     config.put("key3", "author");
     config.put("keyset3", "params");
     config = Collections.unmodifiableMap(config);
 
     MetadataTransform transform = PrebuiltTransforms.deleteMetadata(config);
-    assertEquals("DeleteTransform(keys=[(key=colour,keyset=metadata), "
-        + "(key=author,keyset=params)])", transform.toString());
+    String first = "(key=colour,keyset=metadata)";
```

Comment:
done

i agree that not necessary. i disagree that complicated or more complicated to read. i liked that it was helpful in debugging. done anyway.